owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/logging/LogInterceptor.kt

```diff
@@ -85,7 +85,10 @@ class LogInterceptor : Interceptor {
     private fun logHeaders(headers: Headers): Map<String, String> {
         val auxHeaders = headers.toMap().toMutableMap()
         if (auxHeaders.contains(AUTHORIZATION_HEADER)) {
-            auxHeaders[AUTHORIZATION_HEADER] = "[redacted]"
```

Comment:
I would move this string `"[redacted]" `to the companion object below.

Reply:
This is not going to be used more than once in the class, neither out of this class, so I see no point moving it to a variable. If you check the rest of the class (and in general, the places where we log messages), you can see that we use raw strings in-situ, because they're used just once and they are not translatable